### PR TITLE
chore(ee bridge): fix Redis bridge test flakyness

### PR DESCRIPTION
**Pull request by savonarola**
An attempt to fix flaky tests

### CODE REVIEW

**Review by thalesmg — APPROVED** [review]

**Comment by id** [comment]
@savonarola please rebase on latest master and move the changes to 5.0.14 changelog

File: lib-ee/emqx_ee_bridge/test/emqx_ee_bridge_redis_SUITE.erl [hunk 1]
```diff
@@ -484,8 +492,11 @@ invalid_command_bridge_config() ->
     Conf1 = maps:merge(Conf0, ?COMMON_REDIS_OPTS),
     Conf1#{
         <<"resource_opts">> => #{
-            <<"batch_size">> => <<"1">>,
-            <<"worker_pool_size">> => <<"1">>
+            <<"query_mode">> => <<"sync">>,
+            <<"enable_batch">> => <<"false">>,
+            <<"enable_queue">> => <<"false">>,
```
Comment (thalesmg) [inline lib-ee/emqx_ee_bridge/test/emqx_ee_bridge_redis_SUITE.erl:497]:
```suggestion
            <<"batch_size">> => <<"1">>,
```

File: changes/v5.0.14/feat-9628.en.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Expose additional resource configuration parameters
```
Comment (zmstone) [inline changes/v5.0.14/feat-9628.en.md:1]:
```suggestion
Expose additional resource configuration parameters: `start_after_created` and `start_timeout`.
```

File: changes/v5.0.14/feat-9628.zh.md [hunk 3]
```diff
@@ -0,0 +1 @@
+公开额外的资源配置参数。
```
Comment (zmstone) [inline changes/v5.0.14/feat-9628.zh.md:1]:
```suggestion
为桥接资源增加了配置参数：`start_after_created` 和 `start_timeout`。
```

**Comment by zmstone** [comment]
move the change logs to `v5.0.15`.

**Review by thalesmg — APPROVED** [review]

**Pull request merged**